service/src/main/java/uk/gov/hmcts/reform/fpl/handlers/AdditionalApplicationsUploadedEventHandler.java

```diff
@@ -202,16 +216,20 @@ private void sendNotification(CaseData caseData, Collection<String> emails) {
     @Async
     public void notifyDigitalRepresentatives(final AdditionalApplicationsUploadedEvent event) {
         final CaseData caseData = event.getCaseData();
-        NotifyData notifyData = contentProvider.getNotifyData(caseData);
+        AdditionalApplicationsBundle bundleUploaded = caseData.getAdditionalApplicationsBundle().get(0).getValue();
 
-        Set<String> digitalRepresentativesEmails = getRepresentativesEmails(caseData, DIGITAL_SERVICE);
+        if (bundleUploaded.getC2DocumentBundle() != null || bundleUploaded.getOtherApplicationsBundle() != null) {
```

Comment:
I guess the purpose is to avoid sending notification if a confidential document bundle is uploaded?

Should we check `newBundleUploaded.getC2DocumentBundleConfidential() == null` instead?

Reply:
and shall we add a private method to retrieve the uploaded bundle? Looks like this same logic repeats in several methods.

`AdditionalApplicationsBundle bundleUploaded = caseData.getAdditionalApplicationsBundle().get(0).getValue();`

Reply:
no, not just checking if a confidential C2 is uploaded or not. Have to check if a other application is uploaded as well. User can upload both confidential C2 and other application at the same time.

But yeah, moved the retrieving and checking to a separate method